emsdk.py

```diff
@@ -2775,7 +2770,14 @@ def extract_bool_arg(name):
 
   arg_old = extract_bool_arg('--old')
   arg_uses = extract_bool_arg('--uses')
+  arg_permanent = extract_bool_arg('--permanent')
   arg_global = extract_bool_arg('--global')
+  if arg_global:
+    print('--global is deprecated. Use `--system` to set the environment variables for all users')
+    arg_permanent = True
```

Comment:
I think the old behaviour was the "system" behaviour so this should probably be `arg_system = True` and `arg_permenant = True` and happen after the arg_system handler?

Reply:
https://github.com/emscripten-core/emsdk/pull/621/commits/0f85911cfad108444f97c4e40402a592845fc07e